models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,137 @@ class pp_cond_exp_mc_urbanczik_parameters
+in [1]_. It is capable of connecting to an :doc:`Urbanczik synapse
+<urbanczik_synapse>`.
 
-The model has two compartments: soma and dendrite, labeled as s and p, respectively.
-There is one excitatory and one inhibitory conductance-based synapse onto each compartment.
-Each compartment can also receive current input from a current generator, and an external
-(rheobase) current can be set for each compartment.
+The model has two compartments: soma and dendrite, labeled as s and p,
+respectively. There is one excitatory and one inhibitory conductance-based
+synapse onto each compartment. Each compartment can also receive current input
+from a current generator, and an external (rheobase) current can be set for
+each compartment.
 
 Synapses, including those for injection external currents, are addressed through
 the receptor types given in the receptor_types entry of the state dictionary.
 Note that in contrast to the single-compartment models, all
 synaptic weights must be positive numbers!
 
-
-Parameters:
+.. _multicompartment-models:
+
+Multicompartment models and synaptic delays
++++++++++++++++++++++++++++++++++++++++++++
+
+Note that in case of multicompartment models which represent the dendrite
+explicitly, the interpretation of the synaptic delay in NEST requires a careful
+consideration. In NEST, the delay is at least one simulation time step and is
+assumed to be located entirely at the postsynaptic side. For point neurons, it
+represents the time it takes for an incoming spike to travel along the
+postsynaptic dendrite before it reaches the soma, see panel a). Conversely, if
+the synaptic weight depends on the state of the postsynaptic neuron, the delay
+also represents the time the information on the state propagates back through
+the dendrite to the synapse.
+
+For multicompartment models, this amounts to positioning the delay directly
+behind the incoming synapse, i.e., before the first dendritic compartement on
+the postsynaptic side, see panel b). Therefore, the delay specified in the
+synapse model does *not* account for any delay that might be associated with
+information traveling through the explicitly modeled dendritic compartments.
+
+In case of the Urbanczik synapse, the change of the synaptic weight is driven by
+an error signal which is the difference between the firing rate of the soma
+(derived from the somatic spike train :math:`S_{post}`) and the
+dendritic prediction of it (derived from the dendritic membrane potential
+:math:`V`). The original publication [1]_ does not assume any delay in the
+interaction between the soma and the dendritic compartment. Therefore, we
+compute the error signal from the firing rate and the dendritic prediction at
+equal time points. Due to the synaptic delay :math:`d`, the synapse combines a
+delayed version of the error signal with the presynaptic spike train
+(:math:`S_{pre}`), see panel c).
+
+.. figure:: ../static/img/multicompartment.png
```

Comment:
Can't find this figure, is it already on the git repository or missing from this PR?

Reply:
Hi, the figure should be included in this PR, see /doc/userdoc/static/img/multicompartment.png.